<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Particula</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Destino X:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Destino Y:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Velocidad:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="AgragrInicio_pushButton">
       <property name="text">
        <string>Agregar al Inicio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="opaqueResize">
        <bool>false</bool>
       </property>
       <property name="childrenCollapsible">
        <bool>true</bool>
       </property>
       <widget class="QSpinBox" name="DesX_pinBox">
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="DesY_spinBox_2">
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="Velocidad_spinBox_3">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="Red_spinBox_4">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="Green_spinBox_5">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="Blue_spinBox_6">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="agregarFinal_pushButton">
       <property name="text">
        <string>Agregar al final</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Mostrar_pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="salida">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
